### PR TITLE
rmoving json around merge in provider.tf

### PR DESCRIPTION
**Pull request by PhillypHenning**

### CODE REVIEW

**Comment by LeoDiazL** [comment]
Fixed variables.tf unnecessary default value making tags fail.

File: operations/_scripts/generate/generate_tf_vars.sh [hunk 1]
```diff
@@ -78,6 +78,6 @@ sub_domain_name = \"${SUB_DOMAIN}\"
 
 domain_name = \"${DOMAIN_NAME}\"
 
-additional_tags = {$( IFS=','; echo "${ADDITIONAL_TAGS_LIST[*]}" )}
+additional_tags = \"$( IFS=','; echo "${ADDITIONAL_TAGS_LIST[*]}" )\"
```
Comment (mickmcgrath13) [inline operations/_scripts/generate/generate_tf_vars.sh:81]: what happens here if `ADDITIONAL_TAGS_LIST` is empty?

**Comment by mickmcgrath13** [comment]
@LeoDiazL is this fixing a bug?  seems the aws resources wouldn't be tagged correctly without this change..

**Comment by LeoDiazL** [comment]
Tested in devops-training-ec2-gha-example

**Review by mickmcgrath13 — APPROVED** [review]

**Pull request merged**